DisguiseUnityRenderStream/Runtime/Util/MulticastExtensions.cs

```diff
@@ -51,80 +38,67 @@ public static void EnableMulticast(this UdpClient udpClient, IPAddress multicast
             udpClient.MulticastLoopback = true;
         }
 
-        static (NetworkInterface, UnicastIPAddressInformation) SelectNetworkInterface(string adapterName)
+        public static (string name, IPAddress address) SelectNetworkInterface(string adapterName)
         {
             var nics = NetworkInterface.GetAllNetworkInterfaces();
             NetworkInterface firstUpNic = null;
```

Comment:
Minor: maybe call those two variables: implicitNic?  Or bestNic?  Because it is not necessarily the first one anymore.